File: fizzbuzz/fizzBuzz.py
# FizzBuzz in Python
# by Marshall Ehlinger

def fizzBuzz(max):
	for i in range (1, max + 1):
		line = ""
		if (i % 3 == 0):
			line += 'fizz'
		if (i % 5 == 0):
			line += 'buzz'
		if (line == ""):
			line = str(i)
		print line

########
# TEST #
########

fizzBuzz(100)

File: fizzbuzz/fizzBuzz.r
# FizzBuzz in R
# By Marshall Ehlinger

fizzBuzz <- function(max) {
  for (i in 1:max) {
    line <- "";
    if (i %% 3 == 0) {
      line <- paste(line, "fizz", sep ="");
    }
    if (i %% 5 == 0) {
      line <- paste(line, "buzz", sep ="");
    }
    if (line == "") {
      line <- i;
    }
    print(toString(line));
  }
}

# TEST
fizzBuzz(100);

